docs/examples/classification.ipynb

```diff
@@ -0,0 +1,946 @@
+      "text/plain": [
+       "<Figure size 800x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(8, 8))\n",
+    "plot_reliability_diagram(\n",
```

Comment:
This plot clearly shows that something is wrong with the HGBT model!

Reply:
thanks, is it now perhaps consistent with the ~100% accuracy gradient boosting result that we see on UCI repository? https://archive.ics.uci.edu/dataset/76/nursery 
commit 8d3e7206d7ee6c0c4ab1fa004bdf4dbf33e7a346